docs/vagrant.md

```diff
@@ -22,6 +22,12 @@ Running with Vagrant
     * `splunkforwarder-6.2.5-272645-linux-2.6-x86_64.rpm` and `splunkforwarder-6.2.5-272645-linux-2.6-amd64.deb` ~ 13MB each
 * `vagrant-omnibus` installer currently requires internet access to function.
 
+**Note**: 
+If you want to set up the vagrant cluster to use the license pools defined in the [license pool hash](databags.md#license-hash) databag, add the `configure_guids` recipe to the run_list on the cluster slave (to update the GUIDs on these slaves to predefined values) and update the `license_uri` attribute in the cluster-vagrant databag item to point to the cluster master (_https://33.33.33.30:8089_).
+
+Note that after you spin up the cluster slaves you will have to restart the cluster master to bring the cluster to a stable state. While spinning up the cluster slaves, they are re-assigned with different GUIDS by the `configure_guids` recipe which requires a restart (this restart can take a while to complete). Cluster master is restarted so that it can identify the new GUIDS.
```

Comment:
Nit: This should be part of the previous note, remove 'Note that', capitalize 'After' and get rid of the separation between them.